common/transport/transport-api/src/main/java/org/thingsboard/server/common/transport/util/JsonUtils.java

```diff
@@ -77,4 +88,7 @@ public static JsonObject convertToJsonObject(Map<String, ?> map) {
         return jsonObject;
     }
 
+    public static boolean isBase64(String value) {
```

Comment:
I think you can use: `Base64.isBase64()` from` import org.apache.commons.codec.binary.Base64;`

Reply:
Ok